gno.land/pkg/gnoweb/handler.go

```diff
@@ -120,13 +113,15 @@ func (h *WebHandler) prepareIndexBodyView(r *http.Request, indexData *components
 		return http.StatusNotFound, components.StatusErrorComponent("invalid path")
 	}
 
+	IsHome := IsHomePath(r.RequestURI)
 	breadcrumb := generateBreadcrumbPaths(gnourl)
 	indexData.HeadData.Title = h.Static.Domain + " - " + gnourl.Path
 	indexData.HeaderData = components.HeaderData{
 		Breadcrumb: breadcrumb,
 		RealmURL:   *gnourl,
 		ChainId:    h.Static.ChainId,
 		Remote:     h.Static.RemoteHelp,
+		IsHome:     IsHome,
 	}
```

Comment:
Can you move `IsHome` (and take the opportunity to also move generateBreadcrumb) as it's redundant?

```suggestion
	indexData.HeadData.Title = h.Static.Domain + " - " + gnourl.Path
	indexData.HeaderData = components.HeaderData{
		Breadcrumb: generateBreadcrumbPaths(gnourl),
		RealmURL:   *gnourl,
		ChainId:    h.Static.ChainId,
		Remote:     h.Static.RemoteHelp,
		IsHome:     IsHomePath(r.RequestURI),
	}
```